README.md

```diff
@@ -303,6 +303,15 @@ above and executing exactly the same boot command:
 Shell> fs0:\EFI\BOOT\BOOTX64.EFI
 ```
 
+## How to use on a Jetson nano 4GB ARM board
+
+In Jetson nano, from January 22, 2021, it became possible to save the u-boot to an internal qspi chip. Following the instructions from the first point and specifying the kernel and u-boot versions in the same way as in EVE, we can run it on the Jetson nano with 4GB of RAM.
+
+1. Follow steps in [instruction](https://github.com/Insei/jetson-vanilla-boot/tree/lfedge-eve) for flash jetson boot flow partitions to qspi.
```

Comment:
This looks good, but before we merge this @Insei -- I'd like the content of https://github.com/Insei/jetson-vanilla-boot to be copied under `boards/jetson`. Can you please do it as part of this or separate PR?

Reply:
ok, in eve root? i.e. eve/boards/jetson?